src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -129,14 +132,26 @@ private void addDocument(SortableBundleItem item) throws IOException {
                         currentPageNumber + newDoc.getNumberOfPages());
             }
 
-            if (tableOfContents != null) {
+            if (tableOfContents != null && newDocOutline!= null) {
```

Comment:
formatting issue here, add a space between var and !=

Reply:
done